src/ophyd_async/epics/areadetector/controllers/standard_controller.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+from typing import Optional
+
+from ophyd_async.core import (
+    DEFAULT_TIMEOUT,
+    AsyncStatus,
+    DetectorControl,
+    DetectorTrigger,
+    set_and_wait_for_value,
+    wait_for_value,
+)
+
+from ..drivers.ad_driver import ADDriver, ImageMode
+
+
+class StandardController(DetectorControl):
```

Comment:
Not sure what this is for. It seems areaDetector specific so shouldn't be named "Standard". If we want to share between the different areaDetector drivers then I suggest we abstract the contents of `arm` and `disarm` into `arm_ad_driver_for_frames` and `disarm_ad_driver` utility functions but still keep a `DetectorControl` subclass per sort of detector

Reply:
will rename to ADController for now